rclcpp/test/rclcpp/test_intra_process_manager.cpp

```diff
@@ -23,6 +23,8 @@
 #define RCLCPP_BUILDING_LIBRARY 1
 #include "rclcpp/allocator/allocator_common.hpp"
 #include "rclcpp/macros.hpp"
+#include "rclcpp/serialization.hpp"
```

Comment:
It would be great to add tests verifying that the intraprocess manager is working well with intraprocess messages.